howl/context.py

```diff
@@ -24,30 +24,61 @@ def __init__(self,
                  token_type: str = 'phone',
                  pronounce_dict: PronunciationDictionary = None,
                  use_blank: bool = False):
-        self.labeler = WordFrameLabeler(vocab)
+
         self.coloring = None
-        use_phone = token_type == 'phone'
-        if pronounce_dict is None and use_phone:
-            pronounce_dict = PronunciationDictionary.from_file(SETTINGS.training.phone_dictionary)
-        if use_phone:
+        self.adjusted_vocab = []
+        self.num_labels = 0
+
+        def add_vocab(vocabs: List[str]):
```

Comment:
Maybe pull this out into a class function instead of nesting it inside __init__?